eng/common/pipelines/templates/jobs/perf.yml

```
@@ -7,13 +7,13 @@ parameters:
   default: '360'
 - name: LinuxPool
   type: string
-  default: 'azsdk-pool-mms-ubuntu-2204-perf'
+  default: 'azsdk-pool'
```

```suggestion
  default: 'azsdk-pool-mms-ubuntu-2204-perf'
```

Perf is not changing at this time.